docs/time/performance.inc.html

```diff
@@ -0,0 +1,422 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
I think you didn't mean to commit this file. The html is auto-generated by sphinx so it's not supposed to be in the source code repository.  If you can remove it from the git history that's best but if you're not sure how to do that, you can also just `git rm docs/time/performance.inc.html`  Just be careful not to commit it again!

Reply:
I will remove it from being commited

Reply:
It looks like this was added back?